applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -204,7 +204,6 @@ menuconfig NRF5340_AUDIO_SD_CARD_LC3_STREAMER
 	bool "Audio SD Card LC3 Streamer"
 	depends on NRF5340_AUDIO_SD_CARD_LC3_FILE
 	select EXPERIMENTAL
-	default n
```

Comment:
I believe this should be default n and then set as `y` in the prj.conf?

Reply:
Any kconfig without this will default to `n`, so this is just an unnecessary line